### DISCUSSION TITLE
Problem with smooth scrollbar

### DISCUSSION THREAD

**Discussion by richi-coder** [idiotWu/smooth-scrollbar#550] [Q&A]
I am getting a problem with the lib. It looks like getting the scrollbar.limit.y before completing the page, I am using GSAP too, but I have not encountered any solution

**Comment by sadeghbarati** [comment]
Hi 👋

Can you share a repo?
Which JS framework are you using?

**Comment by richi-coder** [comment]
I am using Vite.js, GSAP and Smooth Scrollbar., implementing a scrollerProxy for GSAP/smooth-scrollbar. Even when getting rid of every gsap scrollTrigger animation I am getting this weird behavior.

I scroll (smooth) to a point, around 80% of the page and suddenly it feels stuck. So, after keeping scrolling in that point it continues to scroll (not being smooth, I guess native scroll) to the end of the page. When scrolling back to hero animations are not playing back (I added scrub: true). It looks misaligned. 

Maybe this behavior is produced by grabbing some html elements with JS, and putting them back to document.body using appendChild(). Smooth-scrollbar is giving me something like: scroller div containing not only a scroll-content div, but also those elements "I added to body". I made this grab and inject to body stuff to make the elements fixed, using position fixed and updating their positions with scrollbar.addListener.

I am working on reproduce the same error in a simpler way to share here.
